session/session.go

```diff
@@ -13,43 +13,35 @@ import (
 // Create attempts to create a session using the node types provided
 // until the first successful creation. If none of the node types are successful, it
 // returns 503 out of capacity error.
-func Create(ctx context.Context, params types.ExecCreateParams, nodeTypeIDs []string) (string, error) {
+func Create(ctx context.Context, params types.ExecCreateParams) (string, error) {
 	uwc := config.InitUnweaveClient()
 
 	var err error
 	var exec *types.Exec
 
-	for _, nodeTypeID := range nodeTypeIDs {
```

Comment:
This should not be deleted. The node type ID here is the `gpuType` eg `a100` vs `rtx_4000`. This iterates over all the nodeTypes the user has configured in their `config.toml`.